run_ci.sh

```diff
@@ -11,8 +11,9 @@ function getlogs(){
     # The logs shared by the ironic containers
     sudo cp -r /opt/dev-scripts/ironic/log $LOGDIR/container-logs
 
-    sudo podman logs coreos-downloader > $LOGDIR/coreos-downloader.log
-    sudo podman logs ipa-downloader > $LOGDIR/ipa-downloader.log
+    for name in ironic ironic-api ironic-conductor ironic-inspector dnsmasq httpd mariadb ipa-downloader coreos-downloader; do
```

Comment:
there is no `ironic` container any more

Reply:
opps, copied that list from 04_setup_ironic.sh where ironic is needed for backwards compat...